app/src/main/java/com/github/se/travelpouch/ui/dashboard/TravelActivityScreen.kt

```diff
@@ -0,0 +1,165 @@
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.outlined.ArrowBack
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.material.icons.filled.DateRange
+import androidx.compose.material.icons.filled.Home
+import androidx.compose.material.icons.filled.Place
+import androidx.compose.material.icons.filled.Settings
+import androidx.compose.material3.Card
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.FloatingActionButton
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.NavigationBar
+import androidx.compose.material3.NavigationBarItem
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.github.se.travelpouch.model.activity.Activity
+import com.github.se.travelpouch.model.activity.ActivityViewModel
+import com.github.se.travelpouch.ui.navigation.NavigationActions
+import java.util.Calendar
+import java.util.GregorianCalendar
+
+/**
+ * This class represents the item for the bottom navigation.
+ *
+ * @property title (String) : the title of the destination in the bottom navigation
+ * @property icon (ImageVector) : the image of the destination.
+ */
+data class BottomNavigationItem(val title: String, val icon: ImageVector)
+
+/**
+ * This function describes how the list of activities of the travel is displayed.
+ *
+ * @param navigationActions (NavigationActions?) : the navigation actions that describes how to go
+ *   from a screen to another.
+ * @param activityModelView (ActivityModelView) : the model view for an activity
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun TravelActivitiesScreen(
+    navigationActions: NavigationActions? = null,
+    activityModelView: ActivityViewModel
+) {
+  // todo: to manually test this function, you need to follow these steps:
+  // comment line 77, decomment the lines 60 to 75, set activityModelView to null, and replace
+  // all the occurrences of listOfActivites.value by activities_test
+  //  val activites_test =
+  //      listOf(
+  //          Activity(
+  //              "1",
+  //              "title1",
+  //              "description1",
+  //              Location(0.0, 0.0, Timestamp(0, 0), "lcoation1"),
+  //              Timestamp(0, 0),
+  //              mapOf<String, Int>()),
+  //          Activity(
+  //              "2",
+  //              "title2",
+  //              "description2",
+  //              Location(0.0, 0.0, Timestamp(0, 0), "lcoation2"),
+  //              Timestamp(0, 0),
+  //              mapOf<String, Int>()))
```

Comment:
```suggestion
```
Here, we shouldn't leave test code once we want to merge it into main. This should stay locally.